src/main/java/io/github/thebusybiscuit/slimefun4/api/items/SlimefunItem.java

```diff
@@ -795,8 +796,10 @@ public void load() {
      *            Any {@link ItemHandler} that should be added to this {@link SlimefunItem}
      */
     public final void addItemHandler(ItemHandler... handlers) {
-        Validate.notEmpty(handlers, "You cannot add zero handlers...");
-        Validate.noNullElements(handlers, "You cannot add any 'null' ItemHandler!");
+        Preconditions.checkArgument(handlers != null && handlers.length > 0, "You cannot add zero handlers...");
+        for (ItemHandler handler : handlers) {
+            Preconditions.checkArgument(handler != null, "You cannot add any 'null' ItemHandler!");
```

Comment:
This should definitely be a util method.

I don't think it warrants needing to be in dough so I'd just add it under our utils package.